sections/experiments/experiments-chapter.tex

```diff
@@ -498,24 +505,186 @@ \subsubsection*{Ablation study for non-contextual recommendation}
+            (20,0.04969)
+            (40,0.04705)
+            (60,0.05145)
+            (80,0.05453)
+            (100,0.05585)
+            (120,0.05453)
+            (140,0.05893)
+            (160,0.06069)
+            (180,0.06157)
+            (200,0.06157)
+        };
+
+        \addplot plot coordinates {
+            (20,0.04749)
+            (40,0.04661)
+            (60,0.04749)
+            (80,0.05101)
+            (100,0.05013)
+            (120,0.05057)
+            (140,0.05365)
+            (160,0.05145)
+            (180,0.05277)
+            (200,0.05497)
+        };
+        \legend{Sideinfo + context,Sideinfo,Context,Nothing}
+        \end{axis}
+    \end{tikzpicture}
+    \caption{The performance on HR@20 of CSGCN-IS on the Yelp-NC dataset with different types of input.}
+    \label{subfig:ablation_study_context_specific_2}
+    \end{subfigure}
+    \caption{Ablation study for the input of CSGCN-ADJ and CSGCN-IS on Yelp-NC.}
+\end{figure*}
+
+\Cref{subfig:ablation_study_context_specific_1} shows the HR@20 for CSGCN-ADJ on YelpNC with different combinations of input. 
+We see that when side-information is included in the input for CSGCN-ADJ it does not change the performance much compared to excluding it.
+Another interesting thing to note is how the model performs worse when context is used in the input, even though we are doing context-specific recommendations.
+This could be because the CSGCN-ADJ model does not actually model context in way that fits the definition of context defined in \Cref{subsec:define_context_sideinfo}.
+In CSGCN-ADJ, context can instead be viewed as a type of side-information, since it is modelled in a somewhat similar way.
+\\\\
+\Cref{subfig:ablation_study_context_specific_2} shows HR@20 for CSGCN-IS on YelpNC with different inputs.
+For CSGCN-IS, there is a slight increase in performance when side-information is included, compared to excluding it.
+This shows that CSGCN-IS is able to use the side-information to better connect user and item nodes, increasing the collaborative signal.
+We also see that for this model, including context in the input does make a significant different for the performance in a context-specific setting.
+This is because context is modelled with finer granularity than in CSGCN-ADJ, meaning that each item and context value combination is represented by a vector.
+The result of this finer granularity, is that context can have different effect on the various items, resulting in more accurate context-specific recommendations.
+In general, we see that side-information can have a slight effect on the results, depending the type of model.
+However, the context seems to have a lot larger effect on the performance of the models, as would be expected for context-aware recommendations.
+Additionally, it may be worth noting that the context used for these experiments is mainly based on discretized values based on timestamps, which may or may not be a useful contextual dimension.
```

Comment:
samme som før. måske en sætning om hvorfor de kunne være skrald

Reply:
Dunno det var @Moesgaarda der tilføjede sætningen så jeg fralægger mig alt ansvar

Reply:
det er ikke min pr

Reply:
hvad fanden

Reply:
jeg stemmer for @Eduardsen ordner det. synes du ikke også det @Moesgaarda??